src/classes.py

```diff
@@ -2,21 +2,53 @@
 # constructor
 
 # YOUR CODE HERE
+
+
+class LatLon:
+    def __init__(self, lat, lon):
+        self.lat = lat
+        self.lon = lon
+
+    def __str__(self):
+        return f'lat: {self.lat}, lon: {self.lon}'
         
 # Make a class Waypoint that can be passed parameters `name`, `lat`, and `lon` to the
 # constructor. It should inherit from LatLon. Look up the `super` method.
 
 # YOUR CODE HERE
 
+class Waypoint(LatLon):
+    def __init__(self, name, lat, lon):
+        self.name = name 
+        super().__init__(lat, lon)
+
+    def __str__(self):
+        return f'name: {self.name}, {super().__str__()}'
+    
+
+
 # Make a class Geocache that can be passed parameters `name`, `difficulty`,
 # `size`, `lat`, and `lon` to the constructor. What should it inherit from?
 
 # YOUR CODE HERE
 
+
+class Geocache(Waypoint):
+    def __init__(self, name, difficulty, size, lat, lon):
+        self.difficulty = difficulty
+        self.size = size
+
+        super().__init__(name, lat, lon)
+
+    def __str__(self):
+        return f'{super().__str__()}, size: {self.size}, difficulty: {self.difficulty}'
+
 # Make a new waypoint and print it out: "Catacombs", 41.70505, -121.51521
 
 # YOUR CODE HERE
 
+waypoint = Waypoint("Catacombs", 41.70505, -121.51521).__str__()
```

Comment:
Just a comment: the '.__str__()' at the end is redundant since print() will call for __str__ method of Waypoint class anyway.  This is only necessary if you just want waypoint to be a string value instead of an instance of Waypoint.  For our purpose, either way works.  Same for line 60.